### DISCUSSION TITLE
Height of the navigation header

### DISCUSSION THREAD

**Discussion by NicolasFabre81** [Budibase/budibase#6457] [Help]
The height of the grey navigation header is quite big.
I could'nt find a way to reduce it.
Is it only possible via CSS? How can this be achieved ? Thanks
![image](https://user-images.githubusercontent.com/28592069/175570766-9d3d71da-ade9-4635-a5ef-ab27da21bd3a.png)

**Comment by melohagan** [comment]
Hi @NicolasFabre81 

What you could do in the *Layouts* tab of the *Design* section is to turn off the navigation bar:
![Screenshot 2022-06-24 at 17 13 03](https://user-images.githubusercontent.com/101575380/175575319-008d9485-d314-4cda-88cd-906bbf6fe1e6.png)

Then you could add in your own custom navigation bar with the height that you want!
